Library/Homebrew/extend/os/linux/extend/ENV/super.rb

```diff
@@ -39,15 +41,16 @@ def homebrew_extra_paths
   end
 
   def determine_extra_rpath_paths
-    paths = ["#{HOMEBREW_PREFIX}/lib"]
+    paths = ["/data/data/com.termux/files/usr/lib"]
+    paths += ["#{HOMEBREW_PREFIX}/lib"]
     paths += run_time_deps.map { |d| d.opt_lib.to_s }
     paths += homebrew_extra_library_paths
     paths
   end
 
   def determine_dynamic_linker_path(formula)
     return "" if formula&.name == "glibc"
-    "#{HOMEBREW_PREFIX}/lib/ld.so"
+    "/system/bin/linker"
```

Comment:
There should be no need for this chunk. `…/lib/ld.so` is a symlink to `/system/bin/linker`.